### ISSUE TITLE
Embed Any Video

### ISSUE THREAD

**Issue by cawoodm** [driftyco/ionic-creator-issues#340]
Currently only YouTube and Vimeo is supported.

Would be nice to embed the URL of any video, even a local one (e.g. /videos/myvid.mp4) which may not exist in creator but will be mixed in later.

**Comment by matthewkremer** [comment]
@cawoodm I'll look into this :) Just curious, any chance the HTML component loads that <video> HTML you shared with me correctly? Could be a short term fix.

**Comment by cawoodm** [comment]
Yep, I could enter and work with direct HTML code. Just couldn't upload the vid. Though I guess most people would have vids hosted on some streaming server... just not us.

```
<video controls>
    <source src="http://www.w3schools.com/html/mov_bbb.mp4">
</video>
```
